src/plugins/condition/components/Condition.vue

```diff
@@ -291,12 +291,12 @@ export default {
         },
         removeCriterion(index) {
             this.condition.configuration.criteria.splice(index, 1);
-            this.persist()
+            this.persist();
         },
         cloneCriterion(index) {
```

Comment:
I think its somewhat odd for the cloning to happen in the vue file. That was already here though so at this point we probably shouldn't change it.